Client/Frontend/Widgets/PhotonActionSheet.swift

```diff
@@ -156,7 +156,7 @@ class PhotonActionSheet: UIViewController, UITableViewDelegate, UITableViewDataS
                 make.width.equalTo(width)
                 make.height.equalTo(PhotonActionSheetUX.CancelButtonHeight)
                 if #available(iOS 11, *) {
-                    make.bottom.equalTo(self.view.safeAreaLayoutGuide.snp.bottom)
+                    make.bottom.equalTo(self.view.safeAreaLayoutGuide.snp.bottom).offset(-PhotonActionSheetUX.Padding)
```

Comment:
I think this will add too much padding on an iPhone X. What do you think? Im not sure if it'll look bad on the iPhone X though.

Reply:
Not sure, I'll drop in a screenshot

Reply:
on the X:
![simulator screen shot - iphone x - 2017-10-25 at 13 00 56](https://user-images.githubusercontent.com/5495083/32012414-c3f627d2-b985-11e7-8dd8-30a584dec09f.png)

Reply:
Safari iPhone X: 
![simulator screen shot - iphone x - 2017-10-25 at 13 05 44](https://user-images.githubusercontent.com/5495083/32012441-d21276e0-b985-11e7-88fa-1dd544bf9e7e.png)